v2.0/backup.md

```diff
@@ -26,9 +26,10 @@ Dependent objects should be backed up at the same time as the objects they depen
 
 Object | Depends On
 -------|-----------
-Table with [foreign key](foreign-key.html) constraints | The table it `REFERENCES` (however, this dependency can be [removed during the restore](restore.html#skip_missing_foreign_keys))
-[Views](views.html) | The tables used in the view's `SELECT` statement
-[Interleaved tables](interleave-in-parent.html) | The parent table in the [interleaved hierarchy](interleave-in-parent.html#interleaved-hierarchy)
+Table with [foreign key](foreign-key.html) constraints | The table it `REFERENCES` (however, this dependency can be [removed during the restore](restore.html#skip_missing_foreign_keys)).
+Table with a [sequence](create-sequence.html) | The sequence.
```

Comment:
There is actually an option `skip_missing_sequences` which works the same way as `skip_missing_foreign_keys`. Sorry for not calling it out better in release notes or otherwise! I think we need to add something to `restore.md` and link to it here, same as for `skip_missing_foreign_keys`.

Mimicking the section for foreign keys, the new section in `restore.md` would be something like:

> #### `skip_missing_sequences`
>
> - **Description**: If you want to restore a table that depends on a sequence but don't want to restore the sequence it references, you can drop the sequence dependency from a table (its sequence-using `DEFAULT` expression) and then have it restored.
> - **Key**: `skip_missing_sequences`
> - **Value**: *No value*
> - **Example**: `WITH skip_missing_sequences`

(Note: it's a bit odd that the `skip_missing_foreign_keys` section of that doc seems to link to its own heading; removed that)